.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: scala
 
-script: "sbt clean coverage test coverageReport"
+script: "sbt ++$TRAVIS_SCALA_VERSION clean coverageEnable test coverageReport"
```

Comment:
- `clean` is not needed,
- `coverage` should work properly now, so `coverageEnable` is not needed